diff --git a/lib/blockchain_double.js b/lib/blockchain_double.js
index 1bef1ca..c48d655 100644
--- a/lib/blockchain_double.js
+++ b/lib/blockchain_double.js
@@ -96,7 +96,8 @@ BlockchainDouble.prototype.initialize = function(accounts, callback) {
           }
         },
         enableHomestead: true,
-        activatePrecompiles: true
+        activatePrecompiles: true,
+        allowUnlimitedContractSize: true
       });
 
       if (options.debug == true) {
diff --git a/package.json b/package.json
index 2a40133..30be2ac 100644
--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "ethereumjs-block": "~1.2.2",
     "ethereumjs-tx": "^1.3.0",
     "ethereumjs-util": "^5.2.0",
-    "ethereumjs-vm": "2.3.5",
+    "ethereumjs-vm": "link:../ethereumjs-vm",
     "ethereumjs-wallet": "~0.6.0",
     "fake-merkle-patricia-tree": "~1.0.1",
     "heap": "~0.2.6",
